src/sardana/spock/ipython_00_10/genutils.py

```diff
@@ -1100,6 +1100,15 @@ def mainloop(shell=None, user_ns=None):
 
 
 def run(user_ns=None):
+
+    # TODO: Temporary solution, available while Taurus3 is being supported.
+    from taurus import tauruscustomsettings
+    from sardana import sardanacustomsettings
+    max_counts = getattr(sardanacustomsettings,
+                         'TAURUS_MAX_DEPRECATION_COUNTS')
```

Comment:
I think that you should give a default value of 0 here, in case someone does not set TAURUS_MAX_DEPRECATION_COUNTS in sardanacustomsettings